RAQUASI88/src/WIN32/retroachievements.cpp

```diff
@@ -185,7 +183,7 @@ void RA_InitShared()
 static HDC main_hdc;
 void RA_InitUI()
 {
-    RA_Init(g_hWnd, RA_QUASI88, RAQUASI88_VERSION);
+    RA_Init(g_hWnd, RA_QUASI88, Q_VERSION);
```

Comment:
Why did this change? Looking at the define for Q_VERSION, it's the short RA version, we want to pass the full version to the toolkit.

Reply:
I thought I had resolved that conflict in favor of the BuildVer version, but I guess I slipped somewhere.

Reply:
Fixed by force push.